### DISCUSSION TITLE
core.itero on Mac. Alt+Enter issue or module not active?

### DISCUSSION THREAD

**Discussion by kaimcbride** [nvim-neorg/neorg#1099] [Q&A] [answered]
I've only recently installed neorg and spent the last few days tracking down the usual bugs that happen on a Mac install (compiler, etc...).

Everything seems to now be working, except for core.itero. 

Terminal: Alacritty
NVIM v0.9.2
Neorg, latest from using vim-plug to install (5.0?)

From watching the YouTube videos I expect that if, for example, I'm making an unordered list and type Alt+Enter at the end of the line it will start the next line indented and with a "- " inserted to continue making the list. 

I am on a MacBook Pro that doesn't have an Alt key, but I tried all iterations of Option+Enter, Cntrl+Enter, Cntrl+Option+Enter with no luck. 
Then to eliminate the keyboard issue I created this in my init.vm to create a 
LocalLeader to do <M-CR>:

inoremap <localleader>\ <M-CR>

and it does go to the next line when invoked, but I don't get the core.itero behavior. 

Here is my neorg setup:

require('neorg').setup {
    load = {
        ["core.defaults"] = {}, -- Loads default behaviour
        ["core.concealer"] = {
            config = {
            icon_preset = "diamond"
            }
        }, -- Adds pretty icons to your documents
        ["core.pivot"] = {}, 
        ["core.export"] = {}, 
        ["core.summary"] = {}, 
        ["core.itero"] = {},
        ["core.promo"] = {}, 
        ["core.dirman"] = { 
            config = {
                workspaces = {
                    proj = "~/notes/proj",
                    area = "~/notes/area",
                    arch = "~/notes/arch",
                    res = "~/notes/res",
                },
                default_workspace = "proj",
            },
        },
    },
}

thanks!

**Comment by max397574** [comment]
this can be a terminal issue when it doesn't recognize certain keys. Nothing we can do about that afaik

**Reply by kaimcbride** [reply]
I see it got cut off in the formatting here, but with my in inoremap I made a local leader keyboard binding to send the Alt+Enter keystroke.

`inoremap <localleader>\ <M-CR>`

And it does do a carriage return when invoked. Also, I have seen others on here say that they are using Alacritty with neorg. 

thanks

**Reply by max397574** [reply, marked as answer]
that definitely won't work.
That's the whole point of the `nore` part of the `inoremap`.
You should just change the key in the core.keybindings config. You can find that in the wiki

**Reply by kaimcbride** [reply]
Thank you!
That's the fix I was looking for. Just changed that.
